### PR TITLE
Implemented test cases NMS_FT 58, 59, 60

### PR DESCRIPTION
**Pull request by kanchannms**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by koshalt** [comment]
ok to test

**Comment by nmsyogesh** [comment]
raised JIRA bug for the this.

https://applab.atlassian.net/browse/NMS-187

File: testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java [hunk 1]
```diff
@@ -783,4 +785,101 @@ public void verifyFT77()
...
+        HttpPost httpPost = createSubscriptionHttpPost("12345678901", "A",
+                "AP",
+				"123456789012545", "10", "childPack");
+        String expectedJsonResponse = createFailureResponseJson("<callingNumber: Invalid>");
+		assertTrue(SimpleHttpClient.execHttpRequest(httpPost,
+				HttpStatus.SC_BAD_REQUEST, expectedJsonResponse,
+				ADMIN_USERNAME, ADMIN_PASSWORD));
+    }
+
+    @Ignore
```
Comment (frankhuster) [inline testing/src/test/java/org/motechproject/nms/testing/it/api/KilkariControllerBundleIT.java]: Please add comment:
//TODO:  https://applab.atlassian.net/browse/NMS-187

**Comment by frankhuster** [comment]
lgtm, one minor fix requested

**Comment by nmsyogesh** [comment]
The JIRA issue raised for this, is at https://applab.atlassian.net/browse/NMS-197

**Comment by frankhuster** [comment]
@nmsyogesh, please add comment including the jira issue above the @Ignore annotation:
//TODO: https://applab.atlassian.net/browse/NMS-197

**Comment by nmsyogesh** [comment]
@frankhuster I have added JIRA issue in TODO  comment.

**Comment by frankhuster** [comment]
please merge

**Comment by srivastavaAbhi** [comment]
Tracked with branch NMS.FT.KILKARICONTROLLERIT.
So closing this request.

**Pull request closed without merging**